### PR TITLE
Mixpanel counter and tracking

### PR DESCRIPTION
**Pull request by habibillah**
add Mixpanel counter and tracking

### CODE REVIEW

**Comment by austinpapp** [comment]
**just a reminder to myself that i need to add the new label in parameters for mixpanel**

File: backend/src/Civix/ApiBundle/Controller/FollowController.php [hunk 1]
```diff
@@ -139,6 +141,9 @@ public function deleteAction(UserFollow $follow)
         $this->getDoctrine()->getManager()->remove($follow);
         $this->getDoctrine()->getManager()->flush($follow);
 
+        $mp = Mixpanel::getInstance($this->container->getParameter('mixpanel_project_token'));
```
Comment (austinpapp) [inline backend/src/Civix/ApiBundle/Controller/FollowController.php]: could this ever throw an error? meaning, if we can't track for whatever reason, i want to make sure we can still execute the action without issue. that error if raised needs to be logged.

Comment (habibillah) [inline backend/src/Civix/ApiBundle/Controller/FollowController.php]: See this several scenarios:

1. MixPanel API wrapper code unavailable (ex. required composer package not installed) :point_right: **will break app**
2. MixPanel service down :point_right: Not break app
3. MixPanel wrong API token :point_right: Not break app
4. No `mixpanel_project_token` parameter in parameters.yml :point_right: **will break app**
4. No MixPanel account :point_right: Not break app
5. MixPanel decline tracking request :point_right: Not break app

So ensure you install MixPanel package from composer and add `mixpanel_project_token` into `parameters.yml`

**Comment by habibillah** [comment]
> **just a reminder to myself that i need to add the new label in parameters for mixpanel**

Not sure how you deployee this app, but there are [several ways](http://symfony.com/doc/current/cookbook/deployment/tools.html#using-build-scripts-and-other-tools) to automatically update the `parameters.yml`. The simple way is mentioned [here](http://symfony.com/blog/new-in-symfony-2-3-interactive-management-of-the-parameters-yml-file).

**Comment by austinpapp** [comment]
@habibillah there is a process on my end that has to be done. see here https://github.com/PowerlineApp/powerline-server/wiki/Adding-new-parameters-to-parameters.yml

this process isn't that flexible so something we will need to circle back to and clean up. suggested welcomed in slack chat

**Comment by shakaran** [comment]
@austinpapp the upgrade to new symfony 2.3.X app in PR (on hold)  https://github.com/PowerlineApp/powerline-server/pull/83 should solve the issue with new parameters:

**Comment by austinpapp** [comment]
@habibillah please rebase

**Comment by habibillah** [comment]
done

**Comment by jterps08** [comment]
@austinpapp @habibillah what's the status with this item?

**Comment by austinpapp** [comment]
i'm only merging bug fixes right now. once we have ided bugs and resolved to a stable backend, we'll add

**Comment by austinpapp** [comment]
cleaning up

**Pull request closed without merging**